fe/fe-core/src/main/java/org/apache/doris/qe/cache/CacheBeProxy.java

```diff
@@ -0,0 +1,167 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.apache.doris.qe.cache;
+
+import org.apache.doris.proto.PCacheStatus;
+import org.apache.doris.proto.PCacheResponse;
+import org.apache.doris.proto.PUpdateCacheRequest;
+import org.apache.doris.proto.PFetchCacheRequest;
+import org.apache.doris.proto.PFetchCacheResult;
+import org.apache.doris.proto.PClearType;
+import org.apache.doris.proto.PClearCacheRequest;
+import org.apache.doris.qe.SimpleScheduler;
+import org.apache.doris.rpc.BackendServiceProxy;
+import org.apache.doris.rpc.RpcException;
+import org.apache.doris.system.Backend;
+import org.apache.doris.common.Status;
+import org.apache.doris.proto.PUniqueId;
+import org.apache.doris.thrift.TNetworkAddress;
+
+import org.apache.doris.thrift.TStatusCode;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.concurrent.Future;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+import java.util.List;
+
+/**
+ * Encapsulates access to BE, including network and other exception handlin
+ */
+public class CacheBeProxy extends CacheProxy {
+    private static final Logger LOG = LogManager.getLogger(CacheBeProxy.class);
+
+    public void updateCache(UpdateCacheRequest request, Status status) {
+        PUniqueId sqlKey = request.sql_key;
+        Backend backend = CacheCoordinator.getInstance().findBackend(sqlKey);
+        if (backend == null) {
+            LOG.warn("update cache can't find backend, sqlKey {}", sqlKey);
+            return;
+        }
+        TNetworkAddress address = new TNetworkAddress(backend.getHost(), backend.getBrpcPort());
+        try {
+            PUpdateCacheRequest updateRequest = request.getRpcRequest();
+            Future<PCacheResponse> future = BackendServiceProxy.getInstance().updateCache(address, updateRequest);
+            PCacheResponse response = future.get(10000,TimeUnit.MICROSECONDS);
```

Comment:
Would better make the `10000` static const value. In the future, we maybe need it to be configurable.